### ISSUE TITLE
Way to request feature

### ISSUE THREAD

**Issue by Atchbo2001** [gingerchicken/como-client#232]
is there a place to request feature

**Comment by gingerchicken** [comment]
So I should honestly create an issue template for requesting features, I will get round to it at some point!

While I have made many changes within this project, it is very much far from its best/being done, so any suggestions would be greatly appreciated.

**Comment by gingerchicken** [comment]
This did start out as a personal project and I really didn't expect other people viewing it (if I am honest) but I should start taking it a little more serious with docs and ease of use so that more people can use my client.

**Comment by Atchbo2001** [comment]
https://www.youtube.com/watch?v=5gk4Ws4xP7w

**Comment by Atchbo2001** [comment]
https://www.youtube.com/watch?v=5gk4Ws4xP7w this client (discontinued) they still sell double cost and no hwid resets but this fuction of recording the script is amazing that is the recommendation as i couldn't find any feature like this that works like this

**Comment by gingerchicken** [comment]
@Atchbo2001 Fair enough, I will look into adding that, though I would recommend you create another issue for this feature to be added since I have tagged this one as a documentation related issue as the original question was "where can I submit a feature".

(Just post an issue with a link to the video and briefly explain how it works)

**Comment by Atchbo2001** [comment]
Alrighty

**Comment by gingerchicken** [comment]
@Atchbo2001 I created a new issue for this request in issue #236, I may not immediately work on it but I should eventually get round to it.
